judo-ui-react/src/main/resources/actor/src/layout/Footer.tsx.hbs

```diff
@@ -1,11 +1,20 @@
-{{> fragment.header.hbs }}
+import { Link as RouterLink } from 'react-router-dom';
+import { Link, Stack, Typography } from '@mui/material';
 
-import { Link, Typography } from '@mui/material';
-
-export function Footer() {
-  return (
-      <Typography id="application-footer" variant="body2" color="text.secondary" align="center">
-        {{ modelName application.name }} - {new Date().getFullYear()}.
-      </Typography>
-  );
-}
+export const Footer = () => (
+  <Stack direction="row" justifyContent="space-between" alignItems="center" sx={ { p: '24px 16px 0px', mt: 'auto' } }>
+    <Typography variant="caption">{{ modelName application.name }} - {new Date().getFullYear()}.</Typography>
+    {/*<Typography variant="caption">&copy; All rights reserved</Typography>*/}
+    {/*<Stack spacing={1.5} direction="row" justifyContent="space-between" alignItems="center">*/}
+    {/*  <Link component={RouterLink} to="#" target="_blank" variant="caption" color="textPrimary">*/}
+    {/*    About us*/}
+    {/*  </Link>*/}
+    {/*  <Link component={RouterLink} to="#" target="_blank" variant="caption" color="textPrimary">*/}
+    {/*    Privacy*/}
+    {/*  </Link>*/}
+    {/*  <Link component={RouterLink} to="#" target="_blank" variant="caption" color="textPrimary">*/}
+    {/*    Terms*/}
+    {/*  </Link>*/}
+    {/*</Stack>*/}
+  </Stack>
```

Comment:
Is this intentional?

Reply:
Yes, as I mentioned in the demo. Will be used if we need to add any of the commented out sections later.